runtime/world.js

```diff
@@ -267,26 +256,17 @@ Before(async () => {
  * cleanup database before each scenario
  * can use @noDBReset in feature file, if you don't need cleanup database before some scenarios
  */
-Before(function(scenario) {
-	try {
-		const {tags} = scenario.pickle;
-		const tagNames = tags.map((tag) => tag.name);
-		if(tagNames.indexOf('@noDBReset') != -1){
-			console.log('\n\nNo DB reset...');
-			return Promise.resolve();
-		}
-		console.log('\n\nResetting the DB...');
-		const output = execSync('npm run setup', { cwd: '../schulcloud-server', stdio: 'pipe' });
-		console.log('Done.');
-} catch (err) {
-		console.error('Cannot reset the DB. Additional Info:')
-		console.warn('stdout: ', err.stdout.toString());
-		console.warn('stderr: ', err.stderr.toString());
-		console.log('signal: ', err.signal);
-		console.log('status: ', err.status);
-}
-	// access output via `output.toString()`
-	return Promise.resolve();
+Before(async function (scenario) {
+	const { tags } = scenario.pickle;
+	const tagNames = tags.map((tag) => tag.name);
+	if (tagNames.indexOf('@noDBReset') != -1) {
+		console.log('\n\nNo DB reset...');
+		return;
+	}
+	console.log('\n\nResetting the DB...');
+	const output = await axios.post('http://localhost:3333/api/management/database/seed');
```

Comment:
what is port 3333?
could use url and port as defined in shared-objects/servers.js